db/seeds.rb

```diff
@@ -6,3 +6,23 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Teacher.delete_all
```

Comment:
Seeds are usually run in order to populate new system with sample data. Sometimes, engineer can forget on which environment engineer is currently on - who knows if it's production, staging, qa or local environment? You can have many tabs open in your terminal. Think about your experiences - sometimes you forget on which tab in terminal you are on and what branch are you currently at 😄

If you have a seeds file that creates sample records and you run it in production - no biggie, you'll just create couple useless records. At best - security issue if they have weak passwords. However, if a seed file essentially begins with **Teacher.delete_all** - you're in for a trouble 😄

I'd advise removing this line from the seeds - just to prevent future Marta troubles 😄 

If you're aiming for re-usability of the seeds, as in being able to run the seeds multiple times, there are other ways to check for the existence of the record before trying to create it - and Rails has some of them ;).